docker-compose.yml

```diff
@@ -40,9 +40,7 @@ services:
       - "redis"
     build:
       context: .
-    volumes:
-      - "./storage:/dropzone"
-    command: bundle exec sidekiq -C config/sidekiq.yml
+      dockerfile: Dockerfile.sidekiq
```

Comment:
man... its like they should have paid attention to decades of file naming convention and used as _extension_ instead a magic filename...

I tend to name these things Sidekiq.dckr, etc...